spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/ConfigUtils.java

```diff
@@ -194,41 +219,34 @@ public static MultipleSourcesContainer processNamedData(List<StrippedSourceConta
 	 * defined order). This method first searches by labels, find the sources, then uses
 	 * these names to find any profile based sources.
 	 */
+	public static MultipleSourcesContainer processLabeledData(List<StrippedSourceContainer> strippedSources,
```

Comment:
this method has change, quite a lot. I make a distinction between "root" sources and "siblings" sources. 

- root sources are the ones that are _not_ profile based. For example `a`, `a-dev` => `a` is a root source, `a-dev` is a sibling source